### ISSUE TITLE
Can you add a color scheme other than pure black.

### ISSUE THREAD

**Issue by l0ke** [aaether32323/Encoding-Guide#1]
Title. Something lighter colored like base16-github from [here](https://github.com/samme/base16-styles)

**Comment by aaether32323** [comment]
Yeah I'm working on this. Thinking of actually migrating to mdbook or bookdown though, so I don't have to mess with css anymore and people can choose whatever color they like.
